### PR TITLE
Address review comments from Ilari Liusvaara

### PR DESCRIPTION
**Pull request by bemasc**

### CODE REVIEW

File: draft-ietf-tls-ctls.md [hunk 1]
```diff
@@ -320,14 +325,15 @@ additional extensions are allowed here.
...
+in the template, the corresponding `ExtensionType` MUST NOT appear here.  The
+`pre_shared_key` `ExtensionType` MUST NOT appear in either list.
 
 > OPEN ISSUE: Are there other extensions that would benefit from special
 treatment, as opposed to hex values.
 
 Static vectors (see {{static-vectors}}):
 
-* `Extension.extension_data` for any extension in `expected_extensions` whose value has fixed length.  This applies only to the corresponding message.
+* `Extension.extension_data` for any extension in `expected_extensions` whose value does not depend on the vector length field.  This applies only to the corresponding message.
```
[the remark this answers is not in the document]
Comment (ekr) [reply draft-ietf-tls-ctls.md:336]: Perhaps "whose value is of constant length and therefore the length field is unnecessary"

[the remark this answers is not in the document]
Comment (bemasc) [reply draft-ietf-tls-ctls.md]: The intent was to capture the case of extension that aren't fixed length, but whose length can be determined without relying on the extension's length field.  (This includes almost all extensions; very few actually rely on the length field for parsing.)

I changed this to "whose value is self-delimiting (e.g., fixed length)".

**Comment by bemasc** [comment]
@ekr Ping.  This is the last thing blocking a refreshed cTLS draft.

**Review by ekr — APPROVED** [review]
Just missed this. LGTM

**Pull request merged**